### PR TITLE
Support one hot

### PR DESCRIPTION
**Pull request by tomkoren21**

### CODE REVIEW

File: onnx2kerastl/constant_layers.py [hunk 1]
```diff
@@ -19,4 +20,15 @@ def convert_constant_of_shape(node, params, layers, lambda_func, node_name, kera
...
         raise NotImplementedError("ConstantOfShape should have a value param")
-    layers[node_name] = np.ones(layers[node.input[0]], dtype=np.int32)*params['value']
\ No newline at end of file
+    layers[node_name] = np.ones(layers[node.input[0]], dtype=np.int32)*params['value']
+
+
+def convert_one_hot(node, params, layers, lambda_func, node_name, keras_name):
+    axis = params.get('axis', -1)
+    layers[node_name] = tf.one_hot(indices=tf.cast(layers[node.input[0]],
+                                                   tf.int64),
```
Comment (doronHarnoy) [inline onnx2kerastl/constant_layers.py:29]: sure `tf.int64` and not `tf.int32`? usually tensorflow params are in float32 or int32

File: test/layers/constants/test_constant.py [hunk 2]
```diff
@@ -2,17 +2,81 @@
...
+
+            name="test-model",
+            inputs=[
+                helper.make_tensor_value_info("indices", TensorProto.FLOAT, named_input),
+            ],
+            outputs=[
+                helper.make_tensor_value_info("one_hot_encoded", TensorProto.INT64, output_shape)
+            ]
+
+        ))
```
Comment (doronHarnoy) [inline test/layers/constants/test_constant.py:70]: Big like!

**Review by doronHarnoy — APPROVED** [review]
Cool!

File: onnx2kerastl/constant_layers.py (hunk 1 above)
Comment (tomkoren21) [reply onnx2kerastl/constant_layers.py:29]: This actually does not effect the output type and is one of the valid types of input. I put it there to align with how Onnx expects it.

**Pull request merged**